src/cf-layout/src/cf-layout.less

```diff
@@ -8,34 +8,40 @@
 //
 
 // Color variables
-// `$color-` variables referenced in comments are from 18F's U.S. Web Design Standards
-// https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
 
 // .block
-@block__bg:                     lighten( #3a8899, 55% );
-@block__border:                 #3a8899;
-@block__border-top:             @block__border;
-@block__border-right:           @block__border;
-@block__border-bottom:          @block__border;
-@block__border-left:            @block__border;
+@block__bg:                     @gray-5;
+@block__border:                 @gray-40;
+@block__border-top:             @gray-40;
+@block__border-right:           @gray-40;
+@block__border-bottom:          @gray-40;
+@block__border-left:            @gray-40;
```

Comment:
Do you want all the individual borders set to something other than `@block__border`? It may be more explicit the way you have it, so could be better.